sonic_data_client/client_test.go

```diff
@@ -349,3 +352,86 @@ func TestNonDbClientGetError(t *testing.T) {
 		t.Errorf("Expected error from NonDbClient.Get, got nil")
 	}
 }
+
+func ReceiveFromZmq(consumer swsscommon.ZmqConsumerStateTable) (bool) {

```

Comment:
>ReceiveFromZmq [](http://example.com/codeflow?start=5&length=14)

Each function need at least function level comment block to explain the intention, arguments, etc.

Why you need sleep?
Why retry help?

Reply:
Fixed